src/app/overlays/actions/send/send.component.ts

```diff
@@ -165,6 +166,21 @@ export type SendOverlayData = {
                                     >
                                 </div>
                             </div>
+                            <div *ngIf="isBns(recipient)">
+                                <div style="display: flex; align-items: center" class="mat-body-1">
+                                    Is this a BNS domain?
+                                    <button
+                                        (click)="getDomainResolvedAddress(recipient)"
+                                        mat-mini-fab
+                                        back-button
+                                        color="primary"
+                                        data-cy="bns-resolve-button"
+                                        style="margin-left: 4px"
+                                    >
+                                        Yes
+                                    </button>
+                                </div>
+                            </div>
```

Comment:
What are the steps to see this?  I've tried to send 1 BAN to `ban_1dzpfrgi8t4byzmdeidh57p14h5jwbursf1t3ztbmeqnqqdcbpgp9x8j3cw6` (mictest) but not seeing any difference on the Send dialog component.